pkg/models/models.go

```diff
@@ -30,3 +30,19 @@ type TestGroup struct {
 	Duration  float64
 	Events    []TestEvent
 }
+
+type DatabaseTestRow struct {
+	PR       string
+	CommitID string
+	TestName string
+	Result   string
+}
+type DatabaseCommitRow struct {
```

Comment:
consider this name
```
##  DbEnvironmentTestsRow represents a row in db table that has finished tests in each environments
type DbEnvironmentTestsRow struct {

```

Reply:
GORM seems to use the convention of making the table the plural of the struct so I have it as DbEnvironmentTest as the  struct and db_enviornment_tests as the table